src/main/java/org/gridsuite/modification/server/entities/elementary/CreateEquipmentEntity.java

```diff
@@ -0,0 +1,65 @@
+/*
+  Copyright (c) 2021, RTE (http://www.rte-france.com)
+  This Source Code Form is subject to the terms of the Mozilla Public
+  License, v. 2.0. If a copy of the MPL was not distributed with this
+  file, You can obtain one at http://mozilla.org/MPL/2.0/.
+ */
+package org.gridsuite.modification.server.entities.elementary;
+
+import lombok.Getter;
+import lombok.NoArgsConstructor;
+import org.gridsuite.modification.server.ModificationType;
+import org.gridsuite.modification.server.dto.EquipmentCreationInfos;
+import org.gridsuite.modification.server.entities.ModificationEntity;
+
+import javax.persistence.Column;
+import javax.persistence.MappedSuperclass;
+import java.util.Set;
+
+/**
+ * @author Franck Lecuyer <franck.lecuyer at rte-france.com>
+ */
+@NoArgsConstructor
+@Getter
+@MappedSuperclass
+public class CreateEquipmentEntity extends ModificationEntity {
+    @Column(name = "equipmentId")
+    private String equipmentId;
+
+    @Column(name = "equipmentName")
+    private String equipmentName;
+
+    @Column(name = "voltageLevelId")
+    private String voltageLevelId;
+
+    @Column(name = "busId")
+    private String busId;
```

Comment:
NodeBreaker topology ?

Reply:
busId here is a generic term for BusBarSection id in node breaker topology, and Bus id in bus breaker topology